common/buildcraft/builders/TileQuarry.java

```diff
@@ -205,7 +205,7 @@ public void updateEntity() {
 			// In this case, since idling() does it anyway, we should return.
 			return;
 		} else if (stage == Stage.MOVING) {
-			int energyUsed = this.getBattery().useEnergy(20, (int) Math.ceil(20 + getBattery().getEnergyStored() / 10), false);
+			int energyUsed = this.getBattery().useEnergy(20, (int) Math.ceil(20D + (double) getBattery().getEnergyStored() / 10), false);
```

Comment:
Are you sure? I don't know where Java applies the casting, so I would change the number to make sure it does a double division (10D or 10.0) or use parentesis around the casting, but if the casting applies before the division I guess it's fine like this.